ios/src/TGSceneUpdate.h

```diff
@@ -0,0 +1,18 @@
+//
+//  TGSceneUpdate.h
+//  tangram
+//
+//  Created by Karim Naaji on 11/05/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+@interface TGSceneUpdate : NSObject
+
+- (id)initWithPath:(NSString *)path value:(NSString *)value;
+
+@property (strong, nonatomic) NSString* path;
```

Comment:
these should be 
`+@property (copy, nonatomic) NSString* path;` due to how NSStrings are implemented under the covers.